docs/guides/10-integrating-react.md

```diff
@@ -0,0 +1,111 @@
+We are working on react bindings that offer a better user experience, meanwhile this guide contains the basic fundamentals of how to integrate Airhship components with [React.js](https://reactjs.org/).
+
+## Web components
+
+A web component is just an HTML tag with some attributes that control its behaviour. Simply include the HTML tag, and edit its properties through attributes or through javascript. 
+
+To render a web component simply include its tag inside the REACT render function:
+
+
+```jsx
+render() {
+  <as-infowindow src="https://thecatsite.com/styles/thecatsite/xenforo/sources/notice3.png"></as-infowindow>
+}
+```
+
+The problems come when it comes to managing the state, either listening to events or synchronizing the attributes.
```

Comment:
Don't state it as a problem.

`The challenge with this integration is to manage the state...`
or
`The integration work is to manage the state...`